wiki/Tournaments/UKCC/UKCC7/en.md

```diff
@@ -0,0 +1,589 @@
+| Round of 16 + Losers Round 1 | 2019-09-28/2019-09-29 |
+| Quarter Finals + Losers Round 2/3 | 2019-10-5/2019-10-6 |
+| Semi Finals + Losers Round 4/5 | 2019-10-12/2019-10-13 |
+| Winners Finals + Losers 6/Semi Final | 2019-10-19/2019-10-20 |
+| Losers Final + GRAND Finals | 2019-10-26/2019-10-27 |
+
+
+
+## Prizes
+
+![Prizes](img/prizes.png)
+
+| Placing | Prize(s)|
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/crown-gold.png "Winner") | 6 months of osu! supporter, a shiny profile badge and a magnificent trophy! |
+| ![Silver Crown](/wiki/shared/crown-silver.png "Second place") | 3 months of osu! supporter and a sparkling medal |
+| ![Bronze Crown](/wiki/shared/crown-bronze.png "Third place") | 1 month of osu supporter and a sparkling medal |
+
+### Badge
+![UKCC 7 Winner](img/badge.png)
+
+
+## Organisation
+
+![Staff List](img/staff.png)
+
+UK is run by many wonderful people, this list doesn't fully represent how many amazing people contribute as we accept people along the way. If you're interested in helping staff future UK Community Cups, please contact a host in the [Discord](https://discord.gg/jYhjTtj)!
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host(s) | ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |
+| Referees | ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973), ![][flag_DE] [Bakugo-](https://osu.ppy.sh/users/4990127), ![][flag_GB] [Domblade](https://osu.ppy.sh/users/6701945), ![][flag_BE] [DraQeu](https://osu.ppy.sh/users/4425520), ![][flag_DE] [GDLenny](https://osu.ppy.sh/users/8406711), ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850), ![][flag_FR] [Mimir](https://osu.ppy.sh/users/7382734), ![][flag_NL] [nik](https://osu.ppy.sh/users/10077264) |
+| Commentators | ![][flag_GB] [sut_son](https://osu.ppy.sh/users/9622163), ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811), ![][flag_GB] [_Takara](https://osu.ppy.sh/users/11599184), ![][flag_GB] [Kingling](https://osu.ppy.sh/users/7010761), ![][flag_GB] [xFluffyo](https://osu.ppy.sh/users/8397226), ![][flag_GB] [Doomsday is bad](https://osu.ppy.sh/users/3481378), ![][flag_GB] [Daiyaa](https://osu.ppy.sh/users/929134), + a lot of the UK community |
+| Streamers | ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972), ![][flag_GB] [_Takara](https://osu.ppy.sh/users/11599184), ![][flag_GB] [Kingling](https://osu.ppy.sh/users/7010761), ![][flag_GB] [xFluffyo](https://osu.ppy.sh/users/8397226), ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850) |
+| Mappoolers | ![][flag_BR] [Dada](https://osu.ppy.sh/users/9119507), ![][flag_US] [\_p0ke\_](https://osu.ppy.sh/users/929134) |
+| Statistics | ![][flag_NO] [Yokespai](https://osu.ppy.sh/users/6399568) |
+| Graphics | ![][flag_GB] [Melonturtle](https://osu.ppy.sh/users/4775283), ![][flag_GB] [Listie](https://osu.ppy.sh/users/4121535) |
```

Comment:
```suggestion
| Graphics | ![][flag_GB] [Melonturtle](https://osu.ppy.sh/users/4775283), ![][flag_GB] [Listie](https://osu.ppy.sh/users/4121535) |
```
Designer, maybe?

Reply:
yeah let's go with "Designer" or "Graphics designer", it should be a role here